tests/test-utilities/mock-apis.ts

```diff
@@ -26,9 +60,66 @@ export const mockWorkflowsApis = (page: Page) => {
     mockGlobalApis(page),
     mockWorkflowsApi(page),
     mockSearchAttributesApi(page),
+    mockWorkflowsCountApi(page),
   ]);
 };
 
 export const mockNamespaceApis = (page: Page) => {
```

Comment:
These are some solid utils.

Reply:
Thanks, I'm hoping we can continue to follow this pattern where we just compose a bunch of mock calls together for each page we need to test. Then you can still import a mock API fn and call it individually if you need to override something.